src/tutorials/0005/04-exercise.md

```diff
@@ -0,0 +1,9 @@
+For this exercise, we've added a plain text file to your IPFS node for you, and its CID is:
+
+```
+'QmWCscor6qWPdx53zEQmZvQvuWQYxx1ARRCXwYVE4s9wzJ'
+```
+
+Using the `cat` function, retrieve the file's contents and return them as a string.
+
+**Hint:** Don't forget to convert the buffered contents of the text file to a string using `.toString('utf8')` before returning your result.
```

Comment:
The point is to convert the content of the file we placed in the node so it's readable by the user in the lesson output. Otherwise, the lesson output will only be a Buffer, which is arguably clunkier. This is something we already do in the MFS tutorial, too.

Reply:
fixed for lesson 7 but left here